src/components/card/card.jsx

```diff
@@ -5,15 +5,15 @@ import _ from "lodash";
 import { Link } from "gatsby";
 
 import { Category } from "../category";
-import { useCategory } from '../../helpers';
+import { classnames, useCategory } from '../../helpers';
 
 import * as styles from "./card.module.scss";
 
-const Card = ({ data }) => {
+const Card = ({ data, className, keyCard }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
   return (
-    <Link className={styles.styledLink} to={`/${_.kebabCase(data.frontmatter.permalink)}`}>
+    <Link className={classnames(styles.styledLink, className)} to={`/${_.kebabCase(data.frontmatter.permalink)}`} keyCard={keyCard}>
```

Comment:
Does `keyCard` exist as a prop in `Link` component?

Reply:
Done